### PR TITLE
Clarify instructions in practice section of DOM Manipulation and Events

### PR DESCRIPTION
**Pull request by scheals**
- [x] You have read [The Odin Projects Contributing Guide](https://github.com/TheOdinProject/curriculum/blob/main/CONTRIBUTING.md).
 - [x] The title summarizes the change and where it happened, for example: "Fixes punctuation in Clean Code lesson".
 - [ ] If the PR is related to an open issue, use a [relevant keyword](https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests) and reference it with the `#` sign and the issue number.
 - [x] You have previewed your markdown (if any) in the [Odin Markdown Preview Tool](https://www.theodinproject.com/lessons/preview)
 - [ ] If changes are requested, please make the changes in a timely manner. After you submit the changes, request another review from the maintainer (top right).

#### 1. Describe the changes made and include why they are necessary or important:

I have changed the wording of the first step of the practice section in [DOM Manipulation and Events of Foundations](https://www.theodinproject.com/paths/foundations/courses/foundations/lessons/dom-manipulation-and-events) so that a user understands that they are supposed to code along but they can tackle the problem on their own. 

This is due to multiple people thinking that they have to create their own solution which is reinforced by wording like "so let's practice!" and "Do the first exercise". Searching through discord it seems like core team intended for the user to code along but nonetheless decide on their own based on Wes Bos giving you an option to pause the video and have a try of your own. There are couple of problems here:

- Plainly not listening carefully enough and missing the pausing moment - this will railroad the user into the code along while being told by TOP to "do the exercise" and "practice" and this makes for a confusing experience. If I am doing an exercise and practicing, why am I coding along? This is not how it's been done here!
- If the user doesn't miss this moment they are going to most likely try their own solution. This because TOP told them to do the exercise and TOP promotes figuring things on your own so they're not going to go through with code along thinking they should not.
- If the user actually tries to create their own solution they are going to be met with plenty of stuff that they might have not used at this point. Discord search mentions problems with new element methods, callbacks, [] selector or `this`. While it might be a good opportunity for struggle and growth question is whether this is the moment where the struggling should be happening and with those concepts. 

If the idea is to avoid a passive code-along maybe the way to go is to ask the user to come up with some pseudocode at the pause mark? Or maybe preface this practice with [this article](https://medium.com/@ryanfleharty/conquering-the-code-along-5ebe37941f70) about not falling into passive code-along?

#### 2. Related Issue

---------

### CODE REVIEW

**Comment by wise-king-sullyman** [comment]
While I don't disagree that this is a big project for students to try and accomplish on their own at this point in the curriculum, I'm usually extremely hesitant to add copy that essentially just tells people to pay attention.

I will discuss with the team what our goals are for this lesson and get back to you on what a better approach might be.

**Comment by wise-king-sullyman** [comment]
@scheals  I've confirmed that the intent is for it to be a code-along, and I agree that this can confuse people somewhat easily.

I don't agree with your approach though. If you would like to edit this PR so that it just says in simple/clear language that it's intended to be a code-along I will approve that, but if you don't want to you (of course) don't have to.

**Review by wise-king-sullyman — CHANGES_REQUESTED** [review]
I'd like to see this just clarify that it's intended to be a code-along.

**Comment by scheals** [comment]
Got it. On it.

**Comment by scheals** [comment]
Changed it. Is it too direct / short? 

I am very sorry for the mess with the commits. Is there a way of fixing this? I made some mistakes with git and that's why it took me so long.

**Comment by wise-king-sullyman** [comment]
I don't think there's any issue with the extra merge commits, but if you'd like I *can* tell you how to get rid of them. I wouldn't say it's something necessary to move forward with this PR though.

**Comment by scheals** [comment]
I should've instead asked whether I should clean up the commits, sorry for that! I just googled to make sure I could find it on my own and surely, there's plenty of info out there. It's just that I read somewhere that unnecessary commits on pull requests aren't exactly the best so I got flustered and assumed I have to clean up. So let's leave this like this as a testament to many of my git babysteps :)

File: foundations/javascript_basics/DOM_manipulation_and_events.md [hunk 1]
```diff
@@ -380,7 +380,7 @@ You can find a more complete list with explanations of each event on [this page]
 
 Manipulating web pages is the primary benefit of the JavaScript language! These techniques are things that you are likely to be messing with _every day_ as a front-end developer, so let's practice!
 
-1. Do the first exercise in Wes Bos's JavaScript30 program by cloning the repo at [https://github.com/wesbos/JavaScript30](https://github.com/wesbos/JavaScript30). Check out the [Video Tutorial](https://www.youtube.com/watch?v=VuN8qwZoego) for instructions on the project.  
+1. Grab the first exercise in Wes Bos's JavaScript30 program by cloning the repo at [https://github.com/wesbos/JavaScript30](https://github.com/wesbos/JavaScript30). Follow the [Video Tutorial](https://www.youtube.com/watch?v=VuN8qwZoego) by coding along.
```
Comment (wise-king-sullyman) [inline foundations/javascript_basics/DOM_manipulation_and_events.md:383]:
```suggestion
1. Grab the first exercise in Wes Bos's JavaScript30 program by cloning the repo at [https://github.com/wesbos/JavaScript30](https://github.com/wesbos/JavaScript30). Code along with the [Video Tutorial](https://www.youtube.com/watch?v=VuN8qwZoego) to build the rest of the exercise. 
```
Suggested rephrasing that I think flows a bit better.

**Review by wise-king-sullyman — CHANGES_REQUESTED** [review]

**Review by wise-king-sullyman — APPROVED** [review]
I think this is a great change, thank you for your contribution!

**Pull request merged**